common/SC_AppleEventLoop.hpp

```diff
@@ -0,0 +1,34 @@
+/************************************************************************
+ *
+ * Copyright 2013 Seth Nickell <snickell@gmail.com>
```

Comment:
Who is Seth Nickell? ;-)

Reply:
not sure! this was in the file before, and you kept it when you added your functions. do you want the copyright assignment then?

Reply:
ah, I see! well, I would be flattered :-)